contrib/networkmanager-openvpn/template.py

```diff
@@ -0,0 +1,39 @@
+pkgname = "networkmanager-openvpn"
+pkgver = "1.12.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--disable-static",
+    "--enable-lto=yes",
```

Comment:
```suggestion
```

this doesn't do anything except maybe replace -flto=thin with -flto (didn't check final order) which is not what we want